api/v1beta1/webhook_test.go

```diff
@@ -175,6 +175,31 @@ var _ = Describe("ExternalDNS admission webhook", func() {
 			Expect(err).ShouldNot(Succeed())
 			Expect(err.Error()).Should(ContainSubstring("credentials secret must be specified when provider type is AWS"))
 		})
+		It("valid RoleARN", func() {
```

Comment:
@alebedev87 this felt a bit awkward putting this in `webhook_test.go`, since it's not technically testing the webhook, but instead CRD validation.

What are your thoughts? Okay here? Or should we move elsewhere?

Reply:
Well, as the current state, the validation IS in the webhook, so my question here doesn't make sense anymore.